atomate/qchem/firetasks/fragmenter.py

```diff
@@ -0,0 +1,227 @@
+# coding: utf-8
+
+from __future__ import division, print_function, unicode_literals, absolute_import
+
+# This module defines a task that returns all fragments of a molecule
+
+from pymatgen.core.structure import Molecule
+from pymatgen.analysis.graphs import MoleculeGraph
+from atomate.utils.utils import env_chk
+from atomate.qchem.database import QChemCalcDb
+from itertools import combinations
+import networkx as nx
+from fireworks import FiretaskBase, FWAction, explicit_serialize
+
+have_babel = True
+try:
+    from pymatgen.io.babel import BabelMolAdaptor
+    import openbabel as ob
+except ImportError:
+    print("Cannot find OpenBabel! Thus, bonds must be provided by the user.")
+    have_babel = False
+
+__author__ = "Samuel Blau"
+__copyright__ = "Copyright 2018, The Materials Project"
+__version__ = "0.1"
+__maintainer__ = "Samuel Blau"
+__email__ = "samblau1@gmail.com"
+__status__ = "Alpha"
+__date__ = "6/13/18"
+__credits__ = "John Dagdelen, Shyam Dwaraknath"
+
+
+@explicit_serialize
+class FragmentMolecule(FiretaskBase):
+    """
+    Find all unique fragments of a molecule
+
+    Optional params:
+        molecule (Molecule):
+        edges (list): List of index pairs that define graph edges, aka molecule bonds
+        max_cores (int): Maximum number of cores to parallelize over. Defaults to 32.
```

Comment:
see my other comment about max_cores and fw_env / env_chk

Reply:
Also if you can document or refer to the place where one can find all the possible dfeinitions for qchem_input_params that would be better